test/assets/module/LaminasTestApiToolsDbApi/src/LaminasTestApiToolsDbApi/V1/Rpc/Artistalbum/ArtistalbumController.php

```
@@ -0,0 +1,8 @@
+<?php
+namespace LaminasTestApiToolsDbApi\V1\Rpc\Artistalbum;
+
+use Laminas\ApiTools\Doctrine\Server\Controller\RpcController;
+
+class ArtistalbumController extends RpcController
```

PHPStan (2.2.7)

[error] Class LaminasTestApiToolsDbApi\V1\Rpc\Artistalbum\ArtistalbumController extends unknown class Laminas\ApiTools\Doctrine\Server\Controller\RpcController.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)